packages/headless/src/controllers/index.ts

```diff
@@ -347,6 +347,7 @@ export type {
   SmartSnippetQuestionsListProps,
   SmartSnippetQuestionsListState,
   SmartSnippetRelatedQuestion,
+  SmartSnippetQuestionsListCore,
```

Comment:
We should ideally not need to export this. Might be fine to copy and paste interfaces to avoid it IMO.

Reply:
When I don't export this here I get an error with the doc generation script saying that SmartSnippetQuestionsListCore has no proper documentation.

Reply:
Yeah, that's because some other interfaces or methods use the core interface. Ignore my comment here though, this change doesn't seem worth the time.